server/src/database/seeding/seed-data.ts

```diff
@@ -22,7 +22,6 @@ export const seedUsers = (): CreateUserDto[] => {
       city: 'Seattle',
       state: 'Washington',
       zip_code: '98101',
-      email_verified: true,
```

Comment:
@esteban-gs actually, why'd you take these out?
we need them in to be able to login (i have a new condition in the /login endpoint since my sendgrid work finished)

Reply:
@sorry, just seeing this. I removed it because this flag should be only updated by the we service. It seems like it controls a user's `verified` state, so not something that the user should be able to pass to the server